client/my-sites/customer-home/cards/launchpad/keep-building.tsx

```diff
@@ -56,6 +57,15 @@ const LaunchpadKeepBuilding = ( { siteSlug }: LaunchpadKeepBuildingProps ): JSX.
 						window.location.assign( `/domains/add/${ siteSlug }` );
 					};
 					break;
+				case 'drive_traffic':
+					actionDispatch = () => {
+						recordTaskClickTracksEvent( task.completed, task.id );
```

Comment:
Not a blocker, but we should add the `checklist_slug` and `context` properties like Nelson does in #78181